access-manager-api/src/main/java/cwms/cda/spi/IdentityProvider.java

```diff
@@ -6,6 +6,7 @@
 import io.swagger.v3.oas.models.security.SecurityScheme;
 
 public interface IdentityProvider {
+    public static final String PRINCIPAL_KEY = "prinicipal";
```

Comment:
the typo here is intentional to match the typo in the cwms db ?

Reply:
prinicipal in the above code has an extra "i" after the n

Reply:
"d'oh", thank you I managed not to notice that.